src/platforms/flutter/migration.mdx

```diff
@@ -3,6 +3,10 @@ title: Migration Guide
 sidebar_order: 1000
 ---
 
+## Migrating From `sentry_flutter` `6.12.x` to `sentry` `6.13.x`
+
+On Flutter < 3.3 we keep running your `appRunner` callback in a custom error zone ([`runZonedGuarded`](https://api.flutter.dev/flutter/dart-async/runZonedGuarded.html)). Starting with Flutter `3.3`, we are using [`PlatFormdispatcher.onError`](https://docs.flutter.dev/testing/errors#errors-not-caught-by-flutter) to automatically record errors not caught by flutter. No code changes are needed for this behaviour.
```

Comment:
I'd pretty much copy the sentence from https://github.com/getsentry/sentry-docs/pull/5661/files#r999008592 after reviewed by @imatwawana 
IMO this is verbose and confusing, and also wrong, since `not caught by flutter` is wrong as well.

Reply:
Oh, i took the "Not caught by flutter" from the flutter documentation of the error callback. https://docs.flutter.dev/testing/errors#errors-not-caught-by-flutter